publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/config/RabbitMqPropertiesConfig.java

```diff
@@ -117,6 +117,9 @@ private void readSpringProperties() {
                 if((rabbitmqInstanceObject.get("channelsCount") != null) ) {
                     rabbitMqProperties.setChannelsCount(Integer.parseInt(rabbitmqInstanceObject.get("channelsCount").asText()));  
                 }
+                if((rabbitmqInstanceObject.get("waitForConfirmsTimeOut") != null) ) {
+                    rabbitMqProperties.setWaitForConfirmsTimeOut(Long.parseLong(rabbitmqInstanceObject.get("waitForConfirmsTimeOut").asText()));
```

Comment:
If parseLong or parseInt (line 118) fails due to invalid number format, line 123 is skipped and an error message is logged. But how caller of readSpringProperties finds out about that fact? Is it OK to continue if line 123 has bee skipped?

Reply:
if it fails with invalid number format  then user will get invalid configuration at the time of publishing message.